apps/web/src/components/HolidayAlert/HolidayAlert.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { useTheme } from "@gc-digital-talent/theme";
+import { AlertImage } from "@gc-digital-talent/ui";
+
+import lightImage from "~/assets/img/Holiday_2023_graphic_Light_mode.webp";
+import darkImage from "~/assets/img/Holiday_2023_graphic_darkmode.webp";
+
+const HolidayAlert = (): JSX.Element | null => {
+  const { mode } = useTheme();
```

Comment:
Sadly, we cannot use this until we fully support dark mode. You will just need to duplicate the images and hide them on their specific modes 😞 

```tsx
<img src={darkImage} data-h2-display="base(none) base:dark(block)" />
<img src={lightImage} data-h2-display="base(block) base:dark(none)" />
```

Reply:
Ah, okay. Hmm, I changed things around like so in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8769/commits/d162ea71827ab1928376da70212f0788a062573a